src/Umbraco.Web.UI.Client/src/common/resources/language.resource.js

```diff
@@ -0,0 +1,64 @@
+﻿/**
+  * @ngdoc service
+  * @name umbraco.resources.languageResource
+  * @description Handles retrieving and updating language data
+  **/
+function languageResource($q, $http, umbRequestHelper) {
```

Comment:
Super tiny @Shazwazza & I will fix it up shortly, you are injecting in $q but never use it, so lets get rid of it